### PR TITLE
new node stats

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: docs/sql/system.txt [hunk 1]
```diff
@@ -446,7 +452,21 @@ process
 | ``process['max_open_file_descriptors']`` | The maximum number of open file descriptors    | ``Long``     |
 |                                          | Crate can use.                                 |              |
 +------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']``                       | Information about the system CPU.              | ``Object``   |
```
Comment (chaudum) [inline docs/sql/system.txt]: `process['probe_timestamp']` missing

File: docs/sql/system.txt [hunk 2]
```diff
@@ -394,6 +394,12 @@ os
 |                         | machine by the hypervisor for other tasks.   |             |
```
Comment (chaudum) [inline docs/sql/system.txt]: `os['probe_timestamp']` missing

File: docs/sql/system.txt [hunk 3]
```diff
@@ -394,6 +394,12 @@ os
...
 +-------------------------+----------------------------------------------+-------------+
 
+The cpu information values are cached for 1s. They might vary from the live terms. 
+Use the probe timestamps instead to get the terms which are measured at the time 
+of the collect. You could get the CPU usage for some period by analysing the start 
+and the end values using the "Probe-CPU" timestamp. Otherwise the usage could be
```
Comment (chaudum) [inline docs/sql/system.txt]: `os['probe_timestamp']` missing instead of "Probe-CPU" timestamp

File: docs/sql/system.txt [hunk 4]
```diff
@@ -446,7 +452,21 @@ process
...
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']['user']``               | The process CPU user time in milliseconds.     | ``Long``     |
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']['system']``             | The process CPU kernel time in milliseconds.   | ``Long``     |
++------------------------------------------+------------------------------------------------+--------------+
 
+The cpu information values are cached for 1s. They might vary from the live terms. 
+Use the probe timestamps instead to get the terms which are measured at the time 
+of the collect. You could get the CPU usage for some period by analysing the start 
+and the end values using the "Probe-CPU" timestamp. Otherwise the usage could be
```
Comment (chaudum) [inline docs/sql/system.txt]: `process['probe_timestamp']` missing instead of "Probe-CPU" timestamp

File: docs/sql/system.txt [hunk 5]
```diff
@@ -446,7 +466,23 @@ process
 | ``process['max_open_file_descriptors']`` | The maximum number of open file descriptors    | ``Long``     |
 |                                          | Crate can use.                                 |              |
 +------------------------------------------+------------------------------------------------+--------------+
+| ``process['probe_timestamp']``           | The system UNIX timestamp at the moment of     | ``Long``     |
+|                                          | the probe collection.                          |              |
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']``                       | Information about the system CPU.              | ``Object``   |
```
Comment (seut) [inline docs/sql/system.txt]: afaik these are process CPU stats not system

**Comment by seut** [comment]
otherwise ack

**Pull request merged**